runtime/onert/backend/cpu/KernelGenerator.cc

```diff
@@ -325,8 +325,8 @@ void KernelGenerator::visit(const ir::operation::Conv2D &node)
     _return_fn = std::move(fn);
     return;
   }
-  const auto ifm_shape = _ctx.at(ifm_index).shape().asFeature(_current_layout);
-  const auto ofm_shape = _ctx.at(ofm_index).shape().asFeature(_current_layout);
+  const auto ifm_shape = _ctx.at(ifm_index).shape().asFeature(ir::Layout::NHWC);
+  const auto ofm_shape = _ctx.at(ofm_index).shape().asFeature(ir::Layout::NHWC);
```

Comment:
(Optional)
How about introducing an (namespace or static) inline variable for circle layout instead of using an enumerator of layout? I think It's more clear than ever.

Reply:
`asFeature(...)`'s parameter will be removed after all caller is updated.